content/vendor/snapshots/disaster-recovery-snapshots.md

```diff
@@ -0,0 +1,36 @@
+---
+date: 2021-01-04
+linktitle: "Disaster Recovery"
+title: Disaster Recovery
+isBeta: true
+weight: 7
+---
+
+In order to setup disaster recovery snapshots, backups should be configured to use an S3 store that exists outside of the cluster.
+This is especially true for [embedded kURL installs](/kotsadm/installing/installing-embedded-cluster/).
+
+## Existing Cluster Restore
+
+Restore must begin with installing a version of Velero compatible with the one that was used to make the snapshot.
+Please refer to Velero documnetion for [installing](https://velero.io/docs/v1.5/basic-install/) and [configuring](https://velero.io/plugins/) the plugins.
+Note that Restic is required and `--restic` flag must be used with `velero install` command.
+
+Once Velero is installed, KOTS CLI can be used to [list backups](/kots-cli/backup-ls/) and [create restores](/kots-cli/restore/)
+
+## Online Embedded Cluster Restore
+
+The procedure for restoring an online embedded cluster is the same as the one for restoring an existing cluster.
```

Comment:
should we mention that when restoring to a clean instance, the installer will have to be run first?

Reply:
also, there's no doc for restoring in an existing cluster? should we link to the restore command?

Reply:
@salahalsaleh it's the section right above.  Maybe it needs more detail?

Reply:
yes, my bad